stable/dask/values.yaml

```diff
@@ -1,22 +1,25 @@
+---
 # nameOverride: dask
 # fullnameOverride: dask
 
 scheduler:
   name: scheduler
   image:
     repository: "daskdev/dask"
-    tag: 1.1.0
+    tag: 1.1.1
     pullPolicy: IfNotPresent
   replicas: 1
+  # serviceType: "ClusterIP"
```

Comment:
If the user removes the "#" characters valid YAML should remain, so take out the spaces after the "#" here. Other comments look good but check.